RecoBTag/TensorFlow/plugins/DeepDoubleXTFJetTagsProducer.cc

```diff
@@ -0,0 +1,230 @@
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/Framework/interface/makeRefToBaseProdFrom.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+#include "DataFormats/BTauReco/interface/JetTag.h"
+
+#include "DataFormats/BTauReco/interface/DeepDoubleXTagInfo.h"
+#include "RecoBTag/FeatureTools/interface/tensor_fillers.h"
+
+#include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
+
+#include <algorithm>
+
+class DeepDoubleXTFJetTagsProducer : public edm::stream::EDProducer<edm::GlobalCache<tensorflow::GraphDef>> {
```

Comment:
given the overlap with DeepDoubleXONNXJetTagsProducer, I think that some common base or template is needed.